test/unit/weets.service.spec.js

```diff
@@ -0,0 +1,21 @@
+const weetService = require('../../app/services/weetsService');
+
+describe('Weets Service', () => {
+  it('Should be an object', () => {
+    expect(typeof weetService).toBe('object');
+  });
+
+  it('Should generate a random number between 0 and 9999', () => {
+    const number = weetService.getRandomNumber();
```

Comment:
This is not part of the card, but since you are adding it, I will see it :smile: : 
We usually do unit testing, and as so, it should not call any external service. Maybe a test database (ideally not even that).
So, here you should mock the http request to avoid testing against an external service. I recommend nock library for that

Reply:
There also are other things about tests , but i will save it to the testing card